exe-unit/components/gsb-http-proxy/tests/test_proxy.rs

```diff
@@ -0,0 +1,98 @@
+use actix_http::header::HeaderMap;
+use actix_web::{web, HttpResponse, Responder};
+use actix_web::{App, HttpServer};
+use chrono::Utc;
+use futures::StreamExt;
+use test_context::test_context;
+use ya_framework_basic::async_drop::DroppableTestContext;
+use ya_framework_basic::log::enable_logs;
+use ya_gsb_http_proxy::http_to_gsb::HttpToGsbProxy;
+use ya_gsb_http_proxy::message::GsbHttpCallMessage;
+use ya_gsb_http_proxy::response::GsbHttpCallResponseEvent;
+use ya_service_bus::typed as bus;
+
+#[test_context(DroppableTestContext)]
+#[serial_test::serial]
```

Comment:
It should be marked as framework-test. We had agreement with Rekuc that tests touching system shouldn't be ran as basic `cargo test`
```suggestion
#[cfg_attr(not(feature = "framework-test"), ignore)]
#[test_context(DroppableTestContext)]
#[serial_test::serial]
```